docs/04_instance-manager.yaml

```diff
@@ -843,6 +851,17 @@ data:
           - Key: KubernetesCluster
             Value: !Ref ClusterName
             PropagateAtLaunch: 'true'
+          {{if .Spec.EKSCFSpec.EKSCFConfiguration.MetricsCollection}}
```

Comment:
It looks like we can avoid templating entirely if we resolve 'all' from inside `getNodeAutoScalingGroupMetrics()` function.
I think if we can avoid templating it would be a cleaner solution.

So, if you can resolve `all` from inside `getNodeAutoScalingGroupMetrics()`, it means you can just pass a parameter to the cloudformation template and then this block becomes:
```
MetricsCollection:
          - Metrics:
            !Ref NodeAutoScalingGroupMetrics
            Granularity: "1Minute"
```

Reply:
We can do that, but that will become hard-coding the list of these Allowed Values:
GroupMinSize
GroupMaxSize
GroupDesiredCapacity
GroupInServiceInstances
GroupPendingInstances
GroupStandbyInstances
GroupTerminatingInstances
GroupTotalInstances
Then later if AWS adds anything new then, we will have to add to code right.

Reply:
I see, so if we do not specify `Metrics:` cloudformation defaults to all metrics.
This makes sense in that case.

Reply:
Yes